sleap/gui/learning/dialog.py

```diff
@@ -485,6 +504,32 @@ def get_every_head_config_data(
                 cfg = scopedkeydict.make_training_config_from_key_val_dict(
                     loaded_cfg_scoped
                 )
+
+                if len(self.labels.tracks) > 0:
+
+                    # for multiclass topdown the class vectors output stride
+                    # should be the max stride
```

Comment:
Nit: Should follow [Google style guide](https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments) (correct punctuation)